README.md

```diff
@@ -1,53 +1,75 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# HTML CSS Javascript Capstone Project
+# HTML CSS JS Capstone Project
 
-> One paragraph statement about the project.
+> Hello, My name is Zain Sadaqat and this is my HTML CSS JS Capstone project. Kindly take a look by visiting the link given below. :)
```

Comment:
- [ ] Please add a descriptive message explaining about your project specification or how it works.

Reply:
Well done here!